src/core/Fieldset/Fieldset.test.tsx

```diff
@@ -0,0 +1,105 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+import React from 'react';
+import { fireEvent, render } from '@testing-library/react';
+import { Fieldset } from './Fieldset';
+import { InputGroup } from '../InputGroup';
+import { ToggleSwitch } from '../ToggleSwitch';
+import '@testing-library/jest-dom/extend-expect';
```

Comment:
Instead of adding this import to every test file, it could be moved to setup:
https://github.com/testing-library/jest-dom#usage
![image](https://user-images.githubusercontent.com/36186912/129033175-038e94d6-9ead-445e-8d04-41ff85986588.png)

Reply:
I tried moving the jest-dom import to setupTest.js but I wasn't able to figure out how to do it because I kept running into this syntax error. Any ideas how to get around this? 
![image](https://user-images.githubusercontent.com/88331924/129068482-345eacd7-d50c-4aaf-9757-d1c43ec24e39.png)

Reply:
This will be fixed in #258